themes/helphub/style.css

```diff
@@ -900,16 +900,99 @@ a:active {
 }
 
 .single-post .single-post-sidebar {
-	background-color: #d8d8d8;
+	background-color: #fafafa;
 	float: left;
 	width: 30%;
-	padding: 1em;
 	box-sizing: border-box;
 	position: relative;
 }
 
+.single-post .single-post-sidebar, .single-post .single-post-sidebar * {
+	box-sizing: border-box;
+}
+
 .single-post-sidebar .sidebar-content {
-	position: fixed;
+	padding-top: 4rem;
+}
+
+.sidebar-content .widget {
+	margin: 0 0 4rem;
+}
+
+.single-post-sidebar .widget-title {
+	font-size: 16px;
+	text-transform: uppercase;
+	letter-spacing: 1px;
+	background-color: transparent;
+	padding: 1rem 12px;
+	margin-bottom: 0;
+	color: inherit;
+	font-weight: 400;
+}
+
+.single-post-sidebar ul {
+	list-style: none;
+}
+
+.single-post-sidebar ul li {
+	border-top: 1px solid #ededed;
+}
+
+.single-post-sidebar ul ul {
+	margin-left: 12px;
+	border-left: 1px solid #ededed;
+}
+
+.single-post-sidebar ul li a {
+	display: block;
+	width: 100%;
+	padding: 8px 40px 8px 8px;
```

Comment:
the padding on `.single-post-sidebar ul li a` could be: `1rem 12px` to match with the `.widget-title`